user_guide.adoc

```diff
@@ -832,6 +832,39 @@ other value which was not listed explicitly.
         _: rec_type_unknown
 ----
 
+If an enumeration has already been defined, you can use references to
+items in the enumeration instead of specifying integers a second time:
+
+[source,yaml]
+----
+seq:
+  - id: key
+    type: u4
+    enum: keys
+  - id: data
+    type:
+      switch-on: key
+      cases:
+        keys::width: data_field_width
+        keys::height: data_field_height
+        keys::depth: data_field_depth
+types:
+  data_field_width:
+    seq:
+      #...
+  data_field_height:
+    seq:
+     #...
+  data_field_depth:
+    seq:
+     #...
```

Comment:
Pedantic person in me cries for that misaligned `#...` ;)
And, anyway, `seq` is totally optional, so may be it's better to wrap it up as:

```yaml
types:
  data_field_width: # ...
  data_field_height: # ...
  data_field_depth: # ...
```

for brevity.